tests/test_telemetry_monitor.py

```diff
@@ -26,6 +26,7 @@
 
 
 import logging
+import os
```

Comment:
super minor: I think you are free to rm `os` here (at least I didn't see it used in the diff)

Reply:
good 👁️ !